swagger.yml

```diff
@@ -594,5 +619,25 @@ definitions:
         type: array
         items:
           $ref: "#/definitions/MeetingInfo"
+    required:
+      - meetings
+
+  MeetingSuggestionObject:
+    type: object
+    properties:
+      meetings:
+        type: array
+        items:
+          $ref: "#/definitions/MeetingInfo"
```

Comment:
Not sure if this is correct, since I only query for start_datetime and end_datetime